multimodal/src/autogluon/multimodal/matcher.py

```diff
@@ -285,6 +279,66 @@ def set_verbosity(self, verbosity: int):
         self._verbosity = verbosity
         set_logger_verbosity(verbosity, logger=logger)
 
+    def _init_pretrained(self):
+
+        if self._config is None:
+            # split out the hyperparameters whose values are complex objects
+            hyperparameters, advanced_hyperparameters = split_hyperparameters(self._hyperparameters)
```

Comment:
advanced_hyperparameters are to support more image augmentations from torchvision transforms (https://pytorch.org/vision/stable/transforms.html#geometry). In this case, the values of hyperparameters are callable objects instead of strings or numbers.